### ISSUE TITLE
Slack app doesn't work with ibus

### ISSUE THREAD

**Issue by tagoh** [flathub/com.slack.Slack#21]
summary says it all. ibus can't activate on slack app at all. even if setting GTK_IM_MODULE=ibus explicitly, slack app seems not communicating to ibus through DBus.

**Comment by chenzhiwei** [comment]
Yes, I also encountered this issue, can someone help on this issue?

I can use ibus in org.gnome.gedit flat app.

cc @nedrichards

**Comment by juhp** [comment]
(Just for reference ibus works fine with Slack rpm desktop app.)

**Comment by juhp** [comment]
`flatpak run --env=GTK_IM_MODULE=xim com.slack.Slack` does work for me though.

**Comment by mrsinham** [comment]
installing with **flatpak** makes ibus works with it. Thanks @juhp for pointing it.
